frontend/packages/kubevirt-plugin/src/components/vms/vm-console.tsx

```diff
@@ -98,3 +161,29 @@ export const VMConsoleFirehose: React.FC<VMTabProps> = ({ obj: vm }) => {
     </div>
   );
 };
+
+type VmConsolesWrapperProps = {
+  vm?: VMKind;
+  vmi?: FirehoseResult<VMIKind>;
+  services?: FirehoseResult<K8sResourceKind[]>;
+  pods?: FirehoseResult<K8sResourceKind[]>;
+  migrations?: FirehoseResult<K8sResourceKind[]>;
+};
+
+type VMConsolesProps = {
+  vm: VMKind;
+  onStartVm: () => void;
+  LoadingComponent: React.FC;
+  vmi?: VMIKind;
+  vnc?: any;
```

Comment:
can you add better types for cnv/serial/rdp ?

Reply:
done